packages/components/package.json

```diff
@@ -10,32 +10,28 @@
   "homepage": "https://github.com/chanzuckerberg/edu-design-system/tree/main/packages/components",
   "license": "MIT",
   "main": "dist/index.js",
-  "publishConfig": {
-    "access": "public",
-    "directory": "dist/",
-    "registry": "https://registry.npmjs.org/"
-  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/chanzuckerberg/edu-design-system.git"
   },
+  "files": ["dist", "tailwind.config.js"],
   "scripts": {
-    "build": "npm run build:clean && npm run build:js && npm run build:styles && npm run build:types && rm -rf dist/playground",
+    "build": "yarn run build:clean && yarn run build:js && yarn run build:styles && yarn run build:types && rm -rf dist/playground",
     "build:clean": "rm -rf dist/",
-    "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/*.spec.ts,src/**/*.spec.tsx --ignore src/**/*.stories.ts,src/**/*.stories.tsx",
+    "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore 'src/**/*.spec.ts','src/**/*.spec.tsx' --ignore 'src/**/*.stories.ts','src/**/*.stories.tsx'",
```

Comment:
Not sure why, but I needed to add some quotes around the paths to make this work right.